src/pages/staticPage.js

```diff
@@ -54,14 +53,8 @@ class StaticPage extends React.Component {
     this.state = {};
```

Comment:
If the state is not used anymore we can remove it (as well as the `constructor`)

Reply:
Done.